app/server/models/transfer_account.py

```diff
@@ -198,7 +198,7 @@ def approve_and_disburse(self, initial_disbursement=None, auto_resolve=False):
         from server.utils.access_control import AccessControl
 
         admin = getattr(g, 'user', None)
-        auto_resolve = initial_disbursement != current_app.config['DEFAULT_INITIAL_DISBURSEMENT']
+        auto_resolve = initial_disbursement == current_app.config['DEFAULT_INITIAL_DISBURSEMENT']
```

Comment:
not sure how this wasn't picked up... 🤷‍♂

Reply:
lol!